test_utils.py

```diff
@@ -80,7 +91,12 @@ def run_specific_tests(container, tests, options=None, fail_fast=True):
 
 
 def run_python_test_suite(container, options=None):
-    # start constructing the run_tests command
+    """Run the entire python test suite for iRODS.
+
+    Arguments:
+    container -- target container on which the test script will run
+    options -- list of strings representing script options to pass to the run_tests.py script
```

Comment:
run_core_tests.py

Reply:
This is referring to `run_tests.py` script which exists in the irods repository under the scripts directory

Reply:
This is as it should be.